imagestreams/golang-centos.json

```diff
@@ -0,0 +1,90 @@
+{
+    "apiVersion": "v1",
+    "kind": "ImageStream",
+    "metadata": {
+        "annotations": {
+            "openshift.io/display-name": "Go"
+        },
+        "name": "golang"
+    },
+    "spec": {
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Build and run Go applications on UBI. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/golang-container/blob/master/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Go available on OpenShift, including major versions updates.",
+                    "iconClass": "icon-go-gopher",
+                    "openshift.io/display-name": "Go (Latest)",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "sampleRepo": "https://github.com/sclorg/golang-ex.git",
+                    "supports": "golang",
+                    "tags": "builder,golang,go"
+                },
+                "from": {
+                    "kind": "ImageStreamTag",
+                    "name": "1.13.4-el7"
```

Comment:
Hm, this community (CentOS) imagestream doesn't look correct -- it links to `1.13.4-el7` but there is no such `1.13.4-el7` tag in this imagestream, only in rhel. When looking more into it, the community build of golang is not in sync with rhel, therefore it's problematic. The last build on dockerhub includes golang 1.10 but it is no longer maintained upstream (which is RHEL here) so not really good candidate. What if we only include `ubi7/go-toolset` instead?

Reply:
Oops, that should have been -ubi8.

Reply:
(a comment not relevant to this particular PR) Hm, it's actually quite valid question for all other UBI7 images -- whether to include centos7 or ubi7 in the community(centos) imagestreams -- we might discuss later, but to not delay these PR any longer, let's go with centos/... images where we have them..

Reply:
As for `1.13.4-el7`, are there no plans for updated go-toolset-centos7?  If not, I could make a "centos" 1.13.4-el7 point to ubi7, just to match the tag naming of the "rhel" image.

Although, that does raise the question, should we just be using ubi7 images in new tags and ditching the RHSCL images entirely (for those where UBI applies)?

Reply:
Guess we're asking the same question at the same time.  With the new style tags, there's really nothing stopping us from using UBI7 instead of centos/rhscl where applicable.  That would also avoid the naming disparity between "el7" and "ubi8" in the same imagestream.  Should we do that?